posts/2024/farmOS-3.2.0/index.md

```diff
@@ -0,0 +1,92 @@
+---
+title: farmOS 3.2.0
+date: 2024-04-10
+author: Michael Stenta
+slug: 2024/farmOS-3.2.0
+categories:
+  - newsletter
+---
+
+# farmOS 3.2.0
+
+We are excited to announce the release of
+[farmOS 3.2.0](https://github.com/farmOS/farmOS/releases/tag/3.2.0)!
+This release includes a number of new features and improvements. Here are some
+of the highlights.
+
+## Comments on farmOS records
+
+If you've ever wanted the ability to have discussions on individual records, now
+you can. Three new modules are available, which add the ability to comment on
+logs, assets, and plans, respectively. Previously, the only way to record these
+kinds of thoughts or conversations was in the notes field of the record, which
+isn't ideal for obvious reasons.
+
+Comments appear at the bottom of each record, and each comment has its own
+author and timestamp, which helps to keep track of who chimed in on the
+conversation and when. It's also possible to reply to comments, so a
+conversation can fork into multiple areas of focus.
+
+## Filter by log notes
+
+On the April 10th, 2024 farmOS
+[monthly call](https://farmos.org/community/monthly-call/), Nick from
```

Comment:
```suggestion
On the [April 10th, 2024 farmOS
monthly call](https://farmos.discourse.group/t/farmos-monthly-call-10-april-2024/1972), Nick from
```

IMO, the forum topic is the better link because it includes more detail about this instance of the call - participants, agenda, etc - and also includes the link about monthly calls in general.